sections/algorithmic_components.tex

```diff
@@ -1155,64 +1168,27 @@ \subsection{Association and Matching}
 \subsubsection{Single CCD to Reference Catalog, Semi-Blind}
 \label{sec:acSingleCCDReferenceMatching}
 
-Given a set of sources detected on a single sensor, and a
-corresponding reference catalog we adopt a simple pattern matching
-algorithm to cross-match between the catalogs.  We assume that the
-sources detected on the sensor have approximate positions given by the
-telescope's initial estimate of a WCS, that we know the plate scale of
-the system, and that positional errors are available for both the
-sensor and reference catalog.  
+Given a set of sources detected on a single sensor, and a corresponding reference catalog, we adopt a simple pattern matching algorithm to cross-match between the catalogs.  We assume that the sources detected on the sensor have approximate positions given by the telescope's initial estimate of a WCS, that we know the plate scale of the system, and that positional errors are available for both the sensor and reference catalog.
 
-Cross-matching is undertaken using the Optimistic Pattern Matching (B) algorithm of Tabur \cite{tabur2007}. The algorithm defines an order $m$ (default 6) size $m-1$ acyclic connected tree as the pattern to match between catalogs. The details of the LSST implementation of the Optimistic Pattern Matching (B) algorithm are as follows:
+Cross-matching may be undertaken using the Optimistic Pattern Matching (B) algorithm of Tabur \cite{tabur2007}. The algorithm defines an order $m$ (default 6) size $m-1$ acyclic connected tree as the pattern to match between catalogs.  Trees are constructed and matched for the brightest $n$ stars (default = 50) in descending order until a match is found.
```

Comment:
Again, just because we are working on this now, things are changing a bit and we might as well take them into account.  I'm not sure we should mention defaults here.  Since they are really implementation details.  If you do want to include them, I think the number of bright stars will be closer to 200 than 50.  I'm not sure what the default number of legs on the acyclic graph is, but we can check.